nbs/03_matcher.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        for key in pattern_attrs.keys():
</pre><p>not bad, but you can make it more elegant:</p><p><a href="https://www.perplexity.ai/search/how-to-write-a-python-func-tha-4hUwhPHFTVy7eLVYELOcGw" target="_blank">https://www.perplexity.ai/search/how-to-write-a-python-func-tha-4hUwhPHFTVy7eLVYELOcGw</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>